### PR TITLE
Fix build when python's DEFAULT_VERSION is 3.x

### PR DESCRIPTION
**Pull request by vishwin**
Refer to FreeBSD Bugzilla PR 223357. This pull request is only for Hacktoberfest accounting purposes, so it may be closed at any time.

### CODE REVIEW

File: sysutils/uefi-edk2-bhyve/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ BUILD_DEPENDS=	bash:shells/bash \
 		${PYTHON_LIBDIR}/lib-dynload/_sqlite3.so:databases/py-sqlite3
 
 USES=		gmake \
-		python:build
+		python:2,build
```
Comment (koobs) [inline sysutils/uefi-edk2-bhyve/Makefile]: `:2` only means depend on the python2 symlink via the lang/python2 port, not "supports or builds with Python 2.x"

This should be `:2.7`

Comment (vishwin) [reply sysutils/uefi-edk2-bhyve/Makefile]: Ah, I had forgotten about a local change I had made in `databases/py-sqlite3`'s Makefile where I explicitly set the `PYTHON_VERSION` to 2.7. This was merely a hack to satisfy certain ports with `USES+=python:2` and had `databases/py-sqlite3` in a depends variable; when `DEFAULT_VERSIONS` contains `python=3.x`, `py27-sqlite3`, which those ports require, will never build. Currently re-testing without my local edit on said other ports.

Comment (vishwin) [reply sysutils/uefi-edk2-bhyve/Makefile]: Anyway, such is beyond the scope of this patch, so closing for now.

**Comment by vishwin** [comment]
Turns out the build actually does fail with python 3.
```
gmake -C Tests
gmake[2]: Entering directory '/wrkdirs/usr/ports/sysutils/uefi-edk2-bhyve/work/uefi-edk2-a36132939e259df79b16699c03c6f1d63c7454b9/BaseTools/Tests'
Traceback (most recent call last):
  File "RunTests.py", line 22, in <module>
    import TestTools
  File "/wrkdirs/usr/ports/sysutils/uefi-edk2-bhyve/work/uefi-edk2-a36132939e259df79b16699c03c6f1d63c7454b9/BaseTools/Tests/TestTools.py", line 87
    print description, '(base64 encoded):'
                    ^
SyntaxError: invalid syntax
gmake[2]: *** [GNUmakefile:17: test] Error 1
gmake[2]: Leaving directory '/wrkdirs/usr/ports/sysutils/uefi-edk2-bhyve/work/uefi-edk2-a36132939e259df79b16699c03c6f1d63c7454b9/BaseTools/Tests'
gmake[1]: *** [GNUmakefile:25: Tests] Error 2
gmake[1]: Leaving directory '/wrkdirs/usr/ports/sysutils/uefi-edk2-bhyve/work/uefi-edk2-a36132939e259df79b16699c03c6f1d63c7454b9/BaseTools'
*** Error code 2

Stop.
make: stopped in /usr/ports/sysutils/uefi-edk2-bhyve
```
Still keeping this pull request closed as the Bugzilla PR is still open.

**Pull request closed without merging**